test/OpenTelemetry.Instrumentation.EventCounters.Tests/EventCountersMetricsTests.cs

```diff
@@ -43,10 +45,9 @@ public void EventCounter()
 
         // Act
         counter.WriteMetric(1997.0202);
-        AwaitExport(meterProvider, metricItems);
+        var metric = AwaitExport(meterProvider, metricItems, "ec.a.c");
```

Comment:
nit: used a named parameter for the string const.
That will make this easier to read/understand.